lib/ctblgrp.gi

```diff
@@ -270,13 +245,11 @@ end;
 
 #############################################################################
 ##
-#F  DxNiceBasis(v,space)
+#F  DxIsInSpace(v,space)
 ##
 DxIsInSpace := function(v,r)
-  if not IsBound(r.space) then
-    r.space:=VectorSpace(Field(r.base[1]),r.base);
-  fi;
-  return v in r.space;
+local nr,nc,m,i,j;
+  return SolutionMat(Matrix(BaseDomain(r.base[1]),r.base),v)<>fail;
```

Comment:
`DxIsInSpace` seems to be the only function that up to now has set the component `space`. After the changes proposed here, `CombinatoricSplit` still tries to access `space` components. Did I  misunderstand something?

Reply:
I think the line in `CombinatoricSplit` is a relic of gap3 code that now never runs. In any case, it should have been a call instead to DxIsInSpace.